dominion/dominion_ui/src/components/CountryPartners/index.js

```diff
@@ -46,10 +47,14 @@ const styles = theme => ({
     [theme.breakpoints.up('md')]: {
       padding: '2rem 1rem'
     }
+  },
+  imageGrayScale: {
+    WebkitFilter: 'grayscale(100%)' /* Safari 6.0 - 9.0 */,
+    filter: 'grayscale(100%)'
```

Comment:
Isn't it better to `grayscale` the logo in Photoshop or GIMP i.e offline? These filters are cool if the image we have to show in `grayscale` in uploaded at runtime or something.